core/src/main/java/com/linecorp/armeria/server/encoding/HttpEncoders.java

```diff
@@ -51,6 +60,16 @@ static DeflaterOutputStream getEncodingOutputStream(HttpEncodingType encodingTyp
                 }
             case DEFLATE:
                 return new DeflaterOutputStream(out, true);
+            case BR:
+                try {
+                    // We use 4 as the default level because it would save more bytes
+                    // than GZIP's default setting and compress data faster.
```

Comment:
```suggestion
                    // We use 4 as the default level because it would save more bytes
                    // than GZIP's default setting and compress data faster.
                    // See: https://blogs.akamai.com/2016/02/understanding-brotlis-potential.html
```